crates/bevy_color/Cargo.toml

```diff
@@ -21,7 +21,7 @@ serde = { version = "1.0", features = [
 thiserror = { version = "2", default-features = false }
 derive_more = { version = "2", default-features = false, features = ["from"] }
 wgpu-types = { version = "26", default-features = false, optional = true }
-encase = { version = "0.11", default-features = false, optional = true }
```

Comment:
This version of `glam` use `glam` 0.30, which is what we want from `bevy_math`, so this upgrade isn't critical.